helm/ph-ee-engine/values.yaml

```diff
@@ -1931,8 +1931,7 @@ keycloak:
 integration_test:
   enabled: false
   replicas: 1
-  image: "docker.io/openmf/ph-ee-integration-test"
-  imageTag: v1.4.0-rc.1
+  image: "docker.io/openmf/ph-ee-integration-test:v1.4.0-rc.1"
```

Comment:
As per our discussion in the morning, this change will not be useful. So please remove changes in this file